config/application.yml.default

```diff
@@ -111,6 +111,8 @@ enable_usps_verification: true
 encrypted_document_storage_enabled: true
 encrypted_document_storage_s3_bucket: 'test-bucket-changeme'
 event_disavowal_expiration_hours: 240
+feature_idv_force_gpo_verification_enabled: false
+feature_idv_hybrid_flow_enabled: true
```

Comment:
I see it's prescribed in the ticket, but I'm not sure I understand the use-case for having this feature flag independent from setting the outage status for phone vendors.

Reply:
After some discussion with @jmhooper , we determined that some on call people would prefer to have a "what I'm disabling switch" in addition to the "what is set as an outage that causes the feature to disable" switch.